tests/dummy/app/router.js

```diff
@@ -14,6 +14,7 @@ Router.map( function() {
     this.route( 'browsers' );
 
     this.resource( 'demos', function() {
+        this.route( 'grid' );
```

Comment:
Yes. The demos/sl-grid is the component demo page, and demos/grid is a larger, stand-alone demonstration of a working grid. We can remove it, but the grid is complicated enough that a larger example seems useful to have.